src/client/Engine/ExternalSources.jsx

```diff
@@ -23,7 +23,7 @@ const ExternalSources = ({ externalSources }) => {
         <OIbTitle label="ExternalSources">
           <p>
             This is a list of dataSources from another OIBus. The name should respect the following convention:
-            OIbusId:datasourceId
+            OIbusId:name
```

Comment:
We are explaining to users so maybe here OIbusId:SourceName maybe a better explanation. of <<OIBUS id>>:<<Source Name>> for example 'OIBus1:HDA'